runner/src/runner/pipeline.clj

```diff
@@ -101,24 +110,19 @@
   {:image     current-image-id
    :mounted   set-of-mounted-resources
    :run-id    run-id
-   :db-client db-client
    :env       env-vars
    :group     pipeline-group
    :name      pipeline-name}
   or a Failure shorting the reduce.
 
   Returns the final build state or a Failure."
-  [{:keys [db-client image run-id env group name]
-    :as build-state} step]
+  [{:keys [database producer] :as config}
```

Comment:
this similarly breaks the tests for exec-step too

Reply:
got it, will also update the docs now for setting up the stream